lib/python/qmk/cli/new/keymap.py

```diff
@@ -0,0 +1,65 @@
+"""
+This script automates the copying of the default keymap into your own keymap.
+"""
+import os
+import sys
+
+from milc import cli
+from qmk.path import keymap, check_directories, create_keymap_directory
+
+
+# TODO(orngshdw): add debugging code if needed:
+#   @cli.argument('--debug', dest='debug', action='store_true', help='specify to turn on debugging messages')
+@cli.entrypoint('Create a new keymap.c from the default keyboard of your choosing')
+def main(cli):
+
+    # get keyboard
+    keyboard = input("Keyboard Name: ")
+    # get username
+    username = input("New Username: ")
+
+    # generate keymap paths
+    kb_path = os.path.abspath(keymap(keyboard))
+    print(kb_path)
+    keymap_path_default = kb_path + "/default"
+    keymap_path = kb_path + "/" + username
+
+    # check directories
+    check_directories(kb_path, keymap_path_default)
+    check_directories(keymap_path, exist=False)
```

Comment:
```suggestion

    if os.path.exists(keymap_path):
        cli.log.error('Keymap %s already exists!', keymap_path)
        exit(1)
```

Reply:
Will we see the cli.log.error when running the script normally? If not is it ok to substitute `exit(1)` with `raise FileExistsError(msg)` or `FileNotFoundError` to communicate the error better?

Reply:
Yes, by default we're printing INFO and higher messages.